ivy/functional/backends/tensorflow/experimental/sorting.py

```diff
@@ -11,3 +11,14 @@ def msort(
     out: Optional[Union[tf.Tensor, tf.Variable]] = None,
 ) -> Union[tf.Tensor, tf.Variable]:
     return tf.sort(a, axis=0)
+
+
+# lexsort
+def lexsort(
+    x: Union[tf.Tensor, tf.Variable, list, tuple],
+    /,
+    *,
+    axis: int = -1,
+    out: Optional[Union[tf.Tensor, tf.Variable]] = None,
+) -> Union[tf.Tensor, tf.Variable]:
+    return tf.lexsort(x, axis)
```

Comment:
By the way, do not forget there is no lexsort in tensorflow. Please try a compositional approach.